### ISSUE TITLE
Track properties dialog does not modify track key

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#2351]
Mixxx 2.0 

For the auto-key detection feature. 

From my original post at the forums:

"I love the new key detection feature, but cannot find a way to UPDATE it, especially if it is wrong. 

Case in point - on track I loaded today has the key listed as "D" (major), Unfortunately, the song is ACTUALLY in E minor. I have tried to update it in the properties tab for the track but alas, it remains in D even after correcting. So one of a few things needs to happen: 1) I'm missing something on updating to the correct key and can fix this; 2) The key detection is somehow permanently changing the key in the library and cannot be changed 3) Is there a way to have the ENTIRE track analyzed closer? I do NOT have fast detection enabled. 4) Something else, and I am open to suggestions"

Bottom line - there needs to be a way to override the key of the song as detected, no differently than if you change the artist name, or corrected a spelling in a track, etc.  

I suppose this has to somehow be backward compatible with the Camelot detection structure as well, but I'll leave that to the coder pros   :)
